f8a_report/main.py

```diff
@@ -38,7 +38,7 @@ def main():
         start_date_wk = (today - timedelta(days=7)).strftime('%Y-%m-%d')
         end_date_wk = today.strftime('%Y-%m-%d')
         r.re_train(start_date_wk, end_date_wk, 'weekly', retrain=True)
-        if os.environ.get('GENERATE_MANIFESTS', 'False') == 'True':
+        if os.environ.get('GENERATE_MANIFESTS', 'False') == 'true':
```

Comment:
[f8a-stacks-report.yaml](https://github.com/openshiftio/saas-analytics/blob/master/bay-services/f8a-stacks-report.yaml#L21) passes values as "True". How come this fix would help?

Reply:
The env variable value on staging shows as true. whatever the case may be, it needs to be tested on dev cluster first before we commit